standard_releases.md

```diff
@@ -88,30 +95,41 @@ least work! The general process is:
   likely already been merged in the course of normal development, but they may
   need additional development, examples adding, etc.
 * Review, test and merge those changes into the `master` branch.
-* Call a "freeze" on the master branch. This freeze lasts during the translation
+* Call a "freeze" on the `master` branch. This freeze lasts during the translation
   process and applies to the schema and docs folders in the repository.
-* Create a new branch from `master` for the translation work. e.g.
-  `0.2.0-translation`
-* Begin translation of the new release (see [translations](/translations))
-* During translation there may be a need to fix issues that exist on the master
+* Begin translation of the new release (see [translations](/translations)) using the
+  BODS-main project in Transifex.
+* During translation there may be a need to fix issues that exist on the `master`
   branch. If they are outside schema and docs they can be developed on a branch
   taken from `master` and merged back in when completed.
   If the changes affect schema and/or docs then they may, in turn, affect the
   translation. The process for making these changes is:
   * First create an issue in Github to track the problem.
   * Then create a new branch that uses the issue number, eg.
     `297-concept-translation`.
-  * When the branch is ready to be merged into master, pull the strings from
-    Transifex and push them to the translation branch eg `0.2.0-translation`.
-  * Now merge the branch (`297-concept-translation`) into master before pushing
-    the updated docs to Transifex.
-  * Check for any changes in Transifex and use the translation branch
-    (`0.2.0-translation`) to replace any translations that have been
-    overwritten.
-  * Once the translation is finalised, merge the translation branch into
-    `master`.
+  * When the branch is ready to be merged into `master`, ensure that the translations
```

Comment:
I've re-thought this and updated it to make it hopefully clearer. The main thing we want to avoid is pushing new source strings to Transifex while translations are in-progress, causing recent translations to be dropped. My updates make this more about communicating with the translators than trying to solve this by wrangling git, which I hope is simpler for everyone, but would like to discuss this.